subsys/lorawan/services/Kconfig

```
@@ -65,6 +65,22 @@ config LORAWAN_FRAG_TRANSPORT
 
 	  The used default port for this service is 201.
 
+choice LORAWAN_FRAG_TRANSPORT_DECODER
+	bool "Fragmented Data Block Transport decoder implementation"
+	default LORAWAN_FRAG_TRANSPORT_DECODER_SEMTECH
```

Looks like you need to add a dependency to fix the CI error:

```suggestion
choice LORAWAN_FRAG_TRANSPORT_DECODER
	bool "Fragmented Data Block Transport decoder implementation"
	depends on LORAWAN_FRAG_TRANSPORT
	default LORAWAN_FRAG_TRANSPORT_DECODER_SEMTECH
```